content/docs/04-clusters/03-edge/04-edgeforge-workflow/03.5-palette-canvos.md

```diff
@@ -0,0 +1,658 @@
+  <br />
+
+
+6. Open the **.arg** file in an editor to change one of the arguments. 
+<br />
+
+  ```bash
+  vi .arg
+  ```
+  <br />
+
+7. In the **.arg** file, change the value of the `CUSTOM_TAG` variable to any lowercase and alphanumeric string. This variable defines the custom image tag for the provider OS images. This guide uses the following custom tag as an example. 
+<br /> 
+
+  ```bash
+  CUSTOM_TAG=palette-learn
+  ```
+  Save the file, and exit the editor. 
+  <br />
+
+8. Issue the command below to save your registration token to a local variable. 
+<br />
+
+  ```bash
+  export token="your_token_here"
+  ```
+  <br />
+  
+9. Copy and issue the command below to create the **user-data** file that includes the tenant registration token. Also, you can click on the *Points of Interest* numbers below to learn more about the main attributes relevant to this example. 
+  <br />
+
+  <PointsOfInterest
+    points={[
+      {
+        x: 260,
+        y: 187,
+        label: 1,
+        description: "Stores the registration token and lets the agent use the auto-registration functionality and authenticate with the provided token.",
+        tooltipPlacement: "rightTop",
+      },
+      {
+        x: 190,
+        y: 262,
+        label: 2,
+        description: "Instructs the installer to turn the host machine off once the installation is complete.",
+      },
+      {
+        x: 190,
+        y: 340,
+        label: 3,
+        description: "Sets the login credentials for Edge hosts. The login credentials will allow you to SSH log into the edge host for debugging purposes.",
+        tooltipPlacement: "rightTop",
+      },
+    ]}
+  >
+
+  ```shell
+  cat << EOF > user-data
+  #cloud-config
+  stylus:
+    site:
+      edgeHostToken: $token
+  install:
+    poweroff: true
+  users:
+    - name: kairos
+      passwd: kairos
+  EOF
+  ```
+
+  </PointsOfInterest>
+  <br />
+
+
+10. CanvOS utility internally uses [Earthly](https://earthly.dev/) to build the target artifacts. Issue the following command to invoke the Earthly build. The `--PE_VERSION` option in the command below signifies the Palette Edge version to use, which refers to the git tag you checked out in the previous step. 
+<br />
+  <WarningBox>
+
+  Ensure to issue the command below with root-level or `sudo` privileges.
+
+  </WarningBox>
+
+  ```shell
+  ./earthly.sh +build-all-images --PE_VERSION=$(git describe --abbrev=0 --tags)
+  ```
+
+  ```bash coloredLines=2-2
+  # Output condensed for readability
+  ===================== Earthly Build SUCCESS ===================== 
+  Share your logs with an Earthly account (experimental)! Register for one at https://ci.earthly.dev.
+  ```
+  This command may take up to 15-20 minutes to finish depending on the resources of the host machine.
+  <br />
+
+
+11. List the Docker images to verify the two provider OS images - one compatible with K3s v1.24.6 and another with K3s v1.25.2.
```

Comment:
```suggestion
11. List the Docker images to review the provider OS images created. By default, provider images for all the Edge-supported Kubernetes versions are created. You can identify the provider images by reviewing the image tag value you used in the  **.arg** file's `CUSTOM_TAG` variable. 
```

CanvOS creates all the images by default, so instead of mentioning two specifically, let's just teach the user to look for images with their tag. For example, you could do something like this

```
docker images --filter=reference='*palette-learn*'
```